ramalama/oci.py

```diff
@@ -116,37 +156,54 @@ def _build(self, source, target, args):
                 c.write(model_car)
             else:
                 c.write(model_raw)
-        run_cmd(
-            [self.conman, "build", "-t", target, "-f", containerfile.name, contextdir], stdout=None, debug=args.debug
+        imageid = (
+            run_cmd([self.conman, "build", "--no-cache", "-q", "-f", containerfile.name, contextdir], debug=args.debug)
+            .stdout.decode("utf-8")
+            .strip()
         )
+        return imageid
+
+    def _create_manifest(self, target, imageid, args):
+        cmd_args = [
+            self.conman,
+            "manifest",
+            "create",
+            "--annotation",
+            f"{annotations.AnnotationModel}=true",
+            "--annotation",
+            f"{ocilabeltype}=''",
+            "--annotation",
+            f"{annotations.AnnotationTitle}=args.SOURCE",
+            target,
+        ]
+        run_cmd(cmd_args, stdout=None, debug=args.debug)
+        run_cmd([self.conman, "manifest", "add", target, imageid], stdout=None, debug=args.debug)
```

Comment:
Can/should we set the artifact-type here?

Reply:
I think I will just list all manifest lists as AI Models via ramalama list when using Docker, and then I can handle annotations differently with Podman.